### PR TITLE
refactor(breadcrumbs): component, make it functional and pluggable #32

### PR DESCRIPTION
**Pull request by nileshgulia1**

### CODE REVIEW

**Review by tiberiuichim — APPROVED** [review]
@nileshgulia1 Nice one.

Let's wait for David's approval and then merge it.

**Review by ichim-david — CHANGES_REQUESTED** [review]
@nileshgulia1 I like what you did as well, but I don't agree with removing the wrapping segment from the breadcrumbs.

If we look at Volto's implementation it has the **breadcrumbs** segment as well which can be used in case you want to have a certain background set  for it.

We don't want to have that dark background set, so we should have a variable for the segment background color to have it transparent or white.

Once you bring back the segment I am fine with the other changes.

File: src/ui/Breadcrumbs/Breadcrumbs.jsx
Comment (nileshgulia1) [inline src/ui/Breadcrumbs/Breadcrumbs.jsx:19]: @ichim-david Hm... I was aware of it, since I don't wanna add additional segment overrides. Anyway, I made the `<Segment/>` as primary and attached.

File: src/ui/Breadcrumbs/Breadcrumbs.jsx
Comment (ichim-david) [inline src/ui/Breadcrumbs/Breadcrumbs.jsx:90]: Bring back size passed to breadcrumb, make it so that a prop can change the size of the breadcrumb.
We need **tiny** which sets size to 12px but other designs might be ok to have simply breadcrumbs which
sets the size to 16px

**Review by ichim-david — CHANGES_REQUESTED** [review]
@nileshgulia1 your changes are fine for Breacrumbs.jsx which was left over as a customization from volto-ims-theme.
The problem is the fact that this package is using Breacrumbs.stories.jsx which has a bunch of usability issues which I've tracked in ticket #32
We will see what we should do after ITML fixes the issues that I have added, perhaps another ticket will need to be added.

**Review by tiberiuichim — COMMENTED** [review]
Regardless of anything, I don't think it's a good idea to place the Breadcrumb in a Container.

Containers in Containers are difficult to manage. Let's not make a design decision that belongs to the implementing website.

**Comment by ichim-david** [comment]
> Regardless of anything, I don't think it's a good idea to place the Breadcrumb in a Container.
> 
> Containers in Containers are difficult to manage. Let's not make a design decision that belongs to the implementing website.
There is a confusion here about which breadcrumbs is used where, have a look at the ticket from taskman for further details

**Comment by tiberiuichim** [comment]
@nileshgulia1 Please don't force push. Try to solve the git problem in other means.

**Comment by ichim-david** [comment]
@nileshgulia1 I'm going to merge this pull request as this work is fine. We will see later the implementation of breadcrumbs  from storybook when ITML fixes the signaled issues

**Pull request merged**
